fastlane/Fastfile

```diff
@@ -114,7 +114,6 @@ platform :ios do
     upload_to_testflight(
         api_key_path: "./ios/ios-fastlane-json-key.json",
         distribute_external: true,
-        reject_build_waiting_for_review: true,
```

Comment:
Should we set this to `false` (even though it's the default) and leave a `TODO` comment to reset to `true` if/when the linked issue is resolved

Reply:
?

Reply:
Sure!